ttkmd_test build fails with g++-15

Ubuntu 24.04.4 + https://launchpad.net/~ubuntu-toolchain-r/+archive/ubuntu/ppa + g++-15, or Ubuntu 25.10.

### Describe the Bug

`make -C test CXX=g++-15` fails.

### Steps to Reproduce

```
sudo add-apt-repository ppa:ubuntu-toolchain-r/ppa
sudo apt update
sudo apt install gcc-15 g++-15
make -C test -j CXX=g++-15
```

### Expected Behavior

ttkmd_test should build without errors.

### Actual Behavior

uintptr_t (not std::uintptr_t) is not found in pin_pages.cpp.

### Kernel Information

N/A

### Driver Information

eb8178ed176935d9f15e4a6981873bbc9363707d

### Hardware Information

N/A

### Relevant Logs and Output

```
pin_pages.cpp: In function ‘void {anonymous}::VerifyPinPagesSimple(const EnumeratedDevice&)’:
pin_pages.cpp:66:57: error: ‘uintptr_t’ does not name a type
   66 |         pin_pages.in.virtual_address = reinterpret_cast<uintptr_t>(page.get());
      |                                                         ^~~~~~~~~
pin_pages.cpp:36:1: note: ‘uintptr_t’ is defined in header ‘<cstdint>’; this is probably fixable by adding ‘#include <cstdint>’
   35 | #include "devfd.h"
  +++ |+#include <cstdint>
   36 |
pin_pages.cpp: In function ‘void {anonymous}::VerifyPinPagesBadFlags(const EnumeratedDevice&)’:
pin_pages.cpp:90:57: error: ‘uintptr_t’ does not name a type
   90 |         pin_pages.in.virtual_address = reinterpret_cast<uintptr_t>(page.get());
      |                                                         ^~~~~~~~~
pin_pages.cpp:90:57: note: ‘uintptr_t’ is defined in header ‘<cstdint>’; this is probably fixable by adding ‘#include <cstdint>’
pin_pages.cpp: In function ‘void {anonymous}::VerifyPinPagesBadSize(const EnumeratedDevice&)’:
pin_pages.cpp:114:57: error: ‘uintptr_t’ does not name a type
  114 |         pin_pages.in.virtual_address = reinterpret_cast<uintptr_t>(page.get());
      |                                                         ^~~~~~~~~
pin_pages.cpp:114:57: note: ‘uintptr_t’ is defined in header ‘<cstdint>’; this is probably fixable by adding ‘#include <cstdint>’
pin_pages.cpp:128:57: error: ‘uintptr_t’ does not name a type
  128 |         pin_pages.in.virtual_address = reinterpret_cast<uintptr_t>(page.get());
      |                                                         ^~~~~~~~~
pin_pages.cpp:128:57: note: ‘uintptr_t’ is defined in header ‘<cstdint>’; this is probably fixable by adding ‘#include <cstdint>’
pin_pages.cpp: In function ‘void {anonymous}::VerifyPinPagesNoUnmapped(const EnumeratedDevice&)’:
pin_pages.cpp:158:57: error: ‘uintptr_t’ does not name a type
  158 |         pin_pages.in.virtual_address = reinterpret_cast<uintptr_t>(mapping.get()) + page_size;
      |                                                         ^~~~~~~~~
pin_pages.cpp:158:57: note: ‘uintptr_t’ is defined in header ‘<cstdint>’; this is probably fixable by adding ‘#include <cstdint>’
pin_pages.cpp:172:57: error: ‘uintptr_t’ does not name a type
  172 |         pin_pages.in.virtual_address = reinterpret_cast<uintptr_t>(mapping.get());
      |                                                         ^~~~~~~~~
pin_pages.cpp:172:57: note: ‘uintptr_t’ is defined in header ‘<cstdint>’; this is probably fixable by adding ‘#include <cstdint>’
pin_pages.cpp: In function ‘void {anonymous}::VerifyPinPagesMultipleRanges(const EnumeratedDevice&)’:
pin_pages.cpp:199:57: error: ‘uintptr_t’ does not name a type
  199 |         pin_pages.in.virtual_address = reinterpret_cast<uintptr_t>(pages.get()) + page_size * i;
      |                                                         ^~~~~~~~~
pin_pages.cpp:199:57: note: ‘uintptr_t’ is defined in header ‘<cstdint>’; this is probably fixable by adding ‘#include <cstdint>’
pin_pages.cpp: In function ‘void {anonymous}::VerifyPinPagesContiguous(const EnumeratedDevice&)’:
pin_pages.cpp:244:65: error: ‘uintptr_t’ does not name a type
  244 |                 pin_pages.in.virtual_address = reinterpret_cast<uintptr_t>(mapping.get());
      |                                                                 ^~~~~~~~~
pin_pages.cpp:244:65: note: ‘uintptr_t’ is defined in header ‘<cstdint>’; this is probably fixable by adding ‘#include <cstdint>’
pin_pages.cpp: In function ‘void {anonymous}::VerifyPinPagesNotContiguous(const EnumeratedDevice&)’:
pin_pages.cpp:314:57: error: ‘uintptr_t’ does not name a type
  314 |         pin_pages.in.virtual_address = reinterpret_cast<uintptr_t>(first_mapping.get());
      |                                                         ^~~~~~~~~
pin_pages.cpp:314:57: note: ‘uintptr_t’ is defined in header ‘<cstdint>’; this is probably fixable by adding ‘#include <cstdint>’
pin_pages.cpp:327:57: error: ‘uintptr_t’ does not name a type
  327 |         pin_pages.in.virtual_address = reinterpret_cast<uintptr_t>(second_mapping.get());
      |                                                         ^~~~~~~~~
pin_pages.cpp:327:57: note: ‘uintptr_t’ is defined in header ‘<cstdint>’; this is probably fixable by adding ‘#include <cstdint>’
pin_pages.cpp: In function ‘void VerifyUnpinPagesSimple(const EnumeratedDevice&)’:
pin_pages.cpp:371:53: error: ‘uintptr_t’ does not name a type
  371 |     pin_pages.in.virtual_address = reinterpret_cast<uintptr_t>(page.get());
      |                                                     ^~~~~~~~~
pin_pages.cpp:371:53: note: ‘uintptr_t’ is defined in header ‘<cstdint>’; this is probably fixable by adding ‘#include <cstdint>’
pin_pages.cpp:378:55: error: ‘uintptr_t’ does not name a type
  378 |     unpin_pages.in.virtual_address = reinterpret_cast<uintptr_t>(page.get());
      |                                                       ^~~~~~~~~
pin_pages.cpp:378:55: note: ‘uintptr_t’ is defined in header ‘<cstdint>’; this is probably fixable by adding ‘#include <cstdint>’
pin_pages.cpp: In function ‘void VerifyUnpinPagesBadSize(const EnumeratedDevice&)’:
pin_pages.cpp:401:53: error: ‘uintptr_t’ does not name a type
  401 |     pin_pages.in.virtual_address = reinterpret_cast<uintptr_t>(page.get());
      |                                                     ^~~~~~~~~
pin_pages.cpp:401:53: note: ‘uintptr_t’ is defined in header ‘<cstdint>’; this is probably fixable by adding ‘#include <cstdint>’
pin_pages.cpp:409:59: error: ‘uintptr_t’ does not name a type
  409 |         unpin_pages.in.virtual_address = reinterpret_cast<uintptr_t>(page.get());
      |                                                           ^~~~~~~~~
pin_pages.cpp:409:59: note: ‘uintptr_t’ is defined in header ‘<cstdint>’; this is probably fixable by adding ‘#include <cstdint>’
pin_pages.cpp:418:59: error: ‘uintptr_t’ does not name a type
  418 |         unpin_pages.in.virtual_address = reinterpret_cast<uintptr_t>(page.get());
      |                                                           ^~~~~~~~~
pin_pages.cpp:418:59: note: ‘uintptr_t’ is defined in header ‘<cstdint>’; this is probably fixable by adding ‘#include <cstdint>’
pin_pages.cpp:427:59: error: ‘uintptr_t’ does not name a type
  427 |         unpin_pages.in.virtual_address = reinterpret_cast<uintptr_t>(page.get());
      |                                                           ^~~~~~~~~
pin_pages.cpp:427:59: note: ‘uintptr_t’ is defined in header ‘<cstdint>’; this is probably fixable by adding ‘#include <cstdint>’
make: *** [Makefile:29: build/pin_pages.o] Error 1
make: *** Waiting for unfinished jobs....
```

### Additional Context

I originally found this on Ubuntu 25.10 without a PPA.

--- a/test/pin_pages.cpp
+++ b/test/pin_pages.cpp
@@ -19,6 +19,7 @@
 #include <cmath>
 #include <cstddef>
 #include <cstdlib>
+#include <cstdint>
 
 #include <sys/types.h>
 #include <sys/stat.h>

